R/get.R

```diff
@@ -13,48 +13,57 @@
 #' time_get(x)
 #' @export
 time_get <- function(time,
-                        components = c("year", "month", "day", "hour", "minute", "second"),
-                        week_start = getOption("timechange.week_start", 1)) {
+                     components = c("year", "month", "yday", "day", "wday", "hour", "minute", "second"),
```

Comment:
Yeh,  pondered just a second over it and decided not to because I didn't want to deal with that `day`, `mday` redundancy. 

To be frank I am slightly more inclined to stick `mday` there for consistency reasons. What is your opinion here?

Reply:
Yep, did that after I left this review